README.md

```diff
@@ -10,18 +10,16 @@ For full documentation of `exec` and `execNode` API, please refer to [`sb-exec R
 
 ```js
 export const FindCache: Map
-class Helpers{
-  static exec(command: String, args: Array<string> = [], options: Object)
-  static execNode(filePath: String, args: Array<string> = [], options: Object)
-  static parse(data: String, regex: String, options: Object = {baseReduction: 1, flags: ""})
-  static rangeFromLineNumber(textEditor: TextEditor, lineNumber: Number, colStart: Number = <firstColumn>):Array
-  static find(directory:String, names: String | Array<string>): ?String
-  static findCached(directory:String, names: String | Array<string>): ?String
-  static findAsync(directory: String, names: String | Array<string>): Promise<?String>
-  static findCachedAsync(directory: String, names: String | Array<string>): Promise<?String>
-  static tempFile<T>(fileName:String, fileContents:String, Callback:Function<T>):Promise<T>
-  static tempFiles<T>(filesNames:Array<{name: String, contents: String}>, callback:Function<T>):Promise<T>
-}
+export function exec(command: String, args: Array<string> = [], options: Object): Promise
+export function execNode(filePath: String, args: Array<string> = [], options: Object): Promise
+export function parse(data: String, regex: String, options: Object = {flags: 'g'}): Array<Linter$Message>
+export function rangeFromLineNumber(textEditor: TextEditor, lineNumber: Number = 0, colStart: Number = <firstTextColumn>):Array
+export function find(directory:String, names: String | Array<string>): ?String
```

Comment:
There should be a linter that does that, I had previously just copy/pasted, I'll tweak them all now